src/Classes/MyRadio/CoreUtils.php

```diff
@@ -858,6 +886,18 @@ public static function getSafeHTML($dirty_html)
     }
 
     /**
+     * Returns lookup values for Status for a select box
+     * @return array
+     */
+    public static function getStatusLookup()
```

Comment:
Potentially Cache this?
We also have a specific form field type for college iirc - is it worth having one for this? I'm tempted to lean toward *not* having one for college and doing it like that.